src/java/fr/paris/lutece/plugins/appointment/web/AbstractAppointmentFormAndSlotJspBean.java

```diff
@@ -362,4 +364,15 @@ public static void addElementsToModel( AppointmentFormDTO appointmentForm, Admin
         model.put( AppointmentUtilities.MARK_PERMISSION_ACCESS_CODE, user.getAccessCode( ) );
     }
 
+    protected boolean checkNbConsecutiveSlots( AppointmentFormDTO appointmentForm )
+    {
+        boolean bReturn = true;
+        if ( appointmentForm.getIsMultislotAppointment( ) && appointmentForm.getNbConsecutiveSlots( ) < 2 )
```

Comment:
Why the restriction of multi-slot calendars to slots with a number of places greater than 2

Reply:
If the condition `appointmentForm.getIsMultislotAppointment()` evaluates to true, it means the user has selected the multi-slot option but entered a number less than 2 for the consecutive slots. In such cases, we set `bReturn `to false and add an appropriate error message using `addError()`.